src/amo/components/AddonReviewCard/index.js

```diff
@@ -294,33 +291,33 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
     const noAuthor = shortByLine || this.isReply();
 
     if (review) {
-      const timestamp = `
-        <a href="/${lang}/${clientApp}/addon/${
-        review.reviewAddon.slug
-      }/reviews/${review.id}/">
-          ${i18n.moment(review.created).fromNow()}
-        </a>
-      `;
+      const url = `/addon/${review.reviewAddon.slug}/reviews/${review.id}/`;
+      const timestamp = (
+        <Link to={url}>{i18n.moment(review.created).fromNow()}</Link>
+      );
+
       const byLineString = noAuthor
         ? // translators: Example in English: "posted last week"
           i18n.gettext('posted %(timestamp)s')
         : // translators: Example in English: "by UserName123, last week"
           i18n.gettext('by %(authorName)s, %(timestamp)s');
 
+      const localized = i18n.sprintf(byLineString, {
+        authorName: review.userName,
+        timestamp: '__timestamp__',
+      });
+
+      const parts = localized.split('__timestamp__');
+      const allParts = [parts.shift(), timestamp, ...parts];
```

Comment:
Similar to my last comment, it seems that `parts.shift()` doesn't have test coverage either.

Reply:
How does that work with RTL?

Reply:
That's a good question. I'm guessing that any localizations done in RTL languages will already be in the correct order, so I think this will "just work".

Reply:
Hmm, we should test this. I think it will probably work, though.

Reply:
I just meant to manually test it. I did so by comparing the Hebrew and Arabic views to the same review listing on -dev. It matches -dev so I think it's working fine.